internal/versions/versions_test.go

```diff
@@ -122,6 +122,30 @@ func TestInstall(t *testing.T) {
 		assertVersionInstalled(t, conf.DataDir, plugin.Name, "1.0.0")
 		assertVersionInstalled(t, conf.DataDir, plugin.Name, "2.0.0")
 	})
+
+	t.Run("if multiple versions are defined and installed returns an error", func(t *testing.T) {
```

Comment:
This is wrong. If if multiple versions are defined we should return successful. I think this should be `if multiple versions are defined and installed returns an error when installing again`

Reply:
Re-read this and realized the description is correct, and so is the test.